client/Containers/Main.jsx

```diff
@@ -11,24 +11,20 @@ import {
 } from 'react-router-dom';
 import AllProducts from './AllProducts';
 import MainNav from './MainNav';
-import FeaturedButton from '../components/styles/FeaturedButton';
+import HomePage from '../Containers/HomePage';
 
 class Main extends Component {
   render() {
     const { fetchProductList } = this.props;
+    console.log('Main', this.props);
     return (
       <Router>
-        <div>
-          <Route component={MainNav} />
+        <MainNav />
```

Comment:
I'm not sure about this, but I think we might want `MainNav` to be a sister node to `Router` rather than a child node